lib/utils/helpers.js

```diff
@@ -1,6 +1,7 @@
 // A collection of useful helper functions, used internally but not exported
 // with the rest of Cylon. 
-"use strict";
+
+// jshint strict:false
```

Comment:
Got rid of this because of the new debounce method, `this` inside a function is a strict mode violation.